### ISSUE TITLE
Failing to unmarshal response

### ISSUE THREAD

**Issue by immesys** [shuheiktgw/go-travis#6]
I have a simple program:

```
	travisclient := travis.NewClient(travis.ApiComUrl, "MYAPIKEY")
	rz, _, err := travisclient.Builds.ListByRepoSlug(context.Background(), "MYSLUG", nil)
	spew.Dump(rz)
	spew.Dump(err)
```

And I am getting some kind of unmarshal error:

```
(json: cannot unmarshal object into Go struct field Build.tag of type string)
```

Is there something I am doing wrong?

Thanks

**Comment by immesys** [comment]
For reference, the tag field it is complaining about looks like this:
```
 "tag": {
        "@type": "tag",
        "@representation": "minimal",
        "repository_id": 7605320,
        "name": "v5.0.11",
        "last_build_id": null
      },
```

So I can see why it doesn't want to parse it into a string

**Comment by shuheiktgw** [comment]
@immesys Thank you for reporting the issue! I think they've changed their api or [the documentation](https://developer.travis-ci.com/resource/builds#collection-items)  was wrong. I need to check their API response.

I'll work on this issue this weekend, so please give me some time or feel free to make a PR! 👍

**Comment by immesys** [comment]
I think their docs are wrong, your code seems to match what the docs say they are giving, but not what is actually returned. I opened a topic on their community forum  (https://travis-ci.community/t/build-tag-api-response-doesnt-match-docs/1907) but nobody has responded yet.

**Comment by shuheiktgw** [comment]
@immesys I confirmed that tag is actually not a string but a struct as you described. So, I fixed the problem and released [v0.1.8](https://github.com/shuheiktgw/go-travis/releases/tag/v0.1.8)!

Thanks again for reporting the issue, and please try the latest version if it works whenever you have time.
